.github/workflows/0-on-demand.yaml

```diff
@@ -64,6 +64,9 @@ env:
 jobs:
   openwhisk:
     runs-on: ubuntu-22.04
+    continue-on-error: false
```

Comment:
Why are we changing the github actions here?

Reply:
I honestly missed this change and I don't really remember 😅
The [actual PR opened to the project](https://github.com/apache/openwhisk/pull/5387/files) didn't have this change.
Sorry for the confusion, this was just a "toy PR" for our discussions, I'll close it.